net/nginx/Makefile

```
@@ -8,12 +8,12 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=nginx
-PKG_VERSION:=1.25.0
+PKG_VERSION:=1.25.1
 PKG_RELEASE:=2
```

```suggestion
PKG_RELEASE:=1
```

Update should be done in separare commit.

---

Done, thanks!